contrib/codegen/funcs/funcs.go

```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 	"text/template"
 
+	"github.com/solo-io/go-utils/contextutils"
```

Comment:
I think we can revert the changes here -- I don't think it's an issue if codegen panics